rllib/agents/ppo/ppo.py

```diff
@@ -95,6 +109,7 @@ def __init__(self):
         self.train_batch_size = 4000
         self.lr = 5e-5
         self.model["vf_share_layers"] = False
+        self._disable_execution_plan_api = True
```

Comment:
Do we need this?

Reply:
Yes, the default is still `False`, so PPOConfig needs to set this here.